appengine_config.py

```diff
@@ -88,8 +88,11 @@ def save(self):
 
 THIRD_PARTY_LIBS = [
     os.path.join(ROOT_PATH, 'third_party', 'backports.functools_lru_cache-1.5'),
-    os.path.join(ROOT_PATH, 'third_party', 'bleach-1.2.2'),
-    os.path.join(ROOT_PATH, 'third_party', 'html5lib-python-0.95'),
+    os.path.join(ROOT_PATH, 'third_party', 'bleach-3.1.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'six-1.12.0'),
```

Comment:
For the new libraries six, python-webencodings and callbacks, could you please reply to this comment and provide links to the documentation that says it is necessary to add these? We should try to avoid adding new third-party libraries where possible, so I want to do due diligence on why we need these ones.

Reply:
Sure. I will get back on this.

Reply:
Hi @seanlip ,
Yes latest version of bleach require these libraries, the same can be found [here](https://github.com/mozilla/bleach/blob/master/CHANGES) (in LN: 23 & 468) and [here](https://github.com/mozilla/bleach/pull/175) (in PR comments). Bleach needs `six` to make the code compatible with python 3.
Also they dint mention about `python-webencoding` and `callbacks` but bleach throws errors in the console if these libraries are not installed because, these libraries are needed for html5lib, the same can be found [here](https://github.com/mozilla/bleach/blob/a06cd773694721f7cace21d09958afdf301f338d/bleach/_vendor/html5lib-1.0.1.dist-info/DESCRIPTION.rst) (in Changelogs).

Reply:
Thanks, @NishealJ! That's good to know, and I appreciate the update.

Reply:
Thanks !